android/titanium/libv8-services.js

```diff
@@ -125,7 +125,11 @@ async function generateSnapshot(v8SnapshotHeaderFilePath, rollupFileContent) {
 		}
 
 		// Delete snapshot blob.
-		await fs.unlink(blobPath);
+		try {
+			await fs.unlink(blobPath);
+		} catch (e) {
+			// Do nothing...
+		}
```

Comment:
I know there's a dozen ways to do this, but now I'm curious, what's the best way to do this?

As I get older, I don't mind calling things and catching the error like you did. It avoids race conditions. There was a stigma that try/catch was performance bottleneck, but that's a thing of the past and this is a CLI we're talking about, not some super hot loop.

Most of the SDK checks if the file exists before deleting like this:

```
		if (fs.existsSync(blobPath)) {
			await fs.unlink(blobPath);
		}
```

However, `fs` here is `fs-extra` which has a handy `fs.remove()` ([docs](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md)) and it silently continues if the file does not exist:

```suggestion
		await fs.remove(blobPath);
```

Reply:
the last one sounds better. I've just copied the existing line as it was already there :smile: Let me check

Reply:
`await fs.remove(blobPath);` works too :+1: 
It's happening when I switch branches and do a `npm run cleanbuild:local`  when I was running a `npm run build:local` before (again: only linux)